lib/kitchen/directions/bake_theorem/v1.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeTheorem
+  class V1
+    def bake(theorem:, number:)
+      theorem['use-subtitle'] = true
+      new_subtitle = theorem.title.cut
+
+      theorem.replace_children(with: %(<div class="os-note-body">#{theorem.children}</div>))
+      note_body = theorem.first('.os-note-body')
```

Comment:
Is `theorem` a type of note? This direction should probably be a variety of `BakeNotes` -- see my comment about notes refactoring